frontend/test/metabase/scenarios/admin/settings/settings.cy.spec.js

```diff
@@ -356,6 +356,17 @@ describe("scenarios > admin > settings", () => {
       .contains(lastItem);
   });
 
+  it("should hide self-hosted settings when running Metabase Cloud", () => {
+    setupMetabaseCloud();
+    cy.visit("/admin/settings/general");
+
+    cy.findByText("Site Name");
+    cy.findByText("Site URL").should("not.exist");
+
+    cy.findByText("Email").should("not.exist");
+    cy.findByText("Updates").should("not.exist");
+  });
+
```

Comment:
You can also use `describeWithToken` helper from `__support__/e2e/cypress` to test the Enterprise tab is also hidden. See an example [here](https://github.com/metabase/metabase/blob/master/frontend/test/metabase/scenarios/admin/audit/ad-hoc.cy.spec.js)

Reply:
Great, thanks!